lib/deployer.js

```diff
@@ -5,14 +5,31 @@ let path     = require('path')
 let netlify  = require('netlify')
 let archiver = require('archiver')
 
+let clean_zip = (res) =>
+	new Promise((resolve, reject) => {
+		fs.readFile(res.zipFile , (err , data) => {
```

Comment:
Are you checking if file exists with this one?
Wouldn't fs.access() be more appropriate for this?

Reply:
Thank you for advicing. You are right. fs.access() be appropriate.